sklearn/utils/estimator_checks.py

```diff
@@ -1027,6 +1026,9 @@ def _check_transformer(name, transformer_orig, X, y):
     else:
         # check for consistent n_samples
         assert X_pred.shape[0] == n_samples
+        if transformer_clone.n_features_out_ is not None:
+            assert X_pred.shape[1] == transformer_clone.n_features_out_
```

Comment:
@adrinjalali You just commented saying you didn't see a test but you had commented on the test earlier?

Reply:
:man_facepalming: I had forgotten about this and was trying to find something in a `test_*.py`, sorry, my bad.

Reply:
> You said all estimators are checked for this, but I don't see a test for the API in common tests.

is what you said. Well this is the line and there's a conversation here you marked as completed ;)